src/Microsoft.OData.Core/UriParser/SemanticAst/SelectExpandClauseExtensions.cs

```diff
@@ -107,12 +109,16 @@ internal static void Traverse<T>(this SelectExpandClause selectExpandClause, Fun
                 T subResult = default(T);
                 if (expandSelectItem.SelectAndExpand.SelectedItems.Any())
                 {
-                    Traverse(expandSelectItem.SelectAndExpand, processSubResult, combineSelectAndExpand,
-                        out subResult);
+                    Traverse(expandSelectItem.SelectAndExpand, processSubResult, combineSelectAndExpand, processApply, out subResult);
                 }
 
-                var expandItem = processSubResult(currentExpandClause, subResult);
+                // TODO: That's PoC we need to handle bunch of cases.

```

Comment:
Please open an issue to track the todo, and add the issue # to the comment.

Reply:
It's handled. Removed the comment

Reply:
Excellent; thanks!

---
In reply to: [321509085](https://github.com/OData/odata.net/pull/1526#discussion_r321509085) [](ancestors = 321509085)